docopt/hash.h

```diff
@@ -14,7 +14,7 @@
  * from requirements by Colin Plumb.
  * (http://burtleburtle.net/bob/hash/doobs.html)
  */
-static inline unsigned int jhash(const char *key, size_t len)
+static INLINE unsigned int jhash(const char *key, size_t len)
```

Comment:
Please no. I've never met c compilers which don't support inline declaration. It also makes possible to include this header in many sources, otherwise you are not able to link those sources in one binary because of multiple definition. So please no.

Reply:
I can remove it… it's just I'm trying to make your project stricter. C89 compatible. No compiler extensions enabled. Pedantic, all warnings enabled, treat warnings as errors.